packages/editor/src/components/editor-help/style.scss

```diff
@@ -1,3 +1,22 @@
+.button {
+	margin: 0.5rem 0.25rem;
```

Comment:
My understanding is that `rem` is not really supported on RN. Technically speaking, the whole of the SCSS pipeline on top of RN is a hack of ours anyway but, I feel that the `rem` is giving a false sense of CSS support.

OTOH, rem is maybe making sense for the web case, but this style file is not used on the web yet. To keep it cross-platform to a good degree, wdyt about defining the margin in terms of the `grid-unit-10` or something?

(Same comment for the `padding` definition below)

Reply:
That sounds good! I'll tweak it. Thanks for the info.